### PR TITLE
[CI:DOCS]Starter for dockerpy regression tests

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: test/dockerpy/images.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import unittest
+import docker
+
+busybox = "docker.io/library/busybox:latest"
+
+def get_client():
+   return docker.DockerClient(base_url="unix://var/run/docker.sock")
```
Comment (jwhonce) [inline test/dockerpy/images.py:7]:
```suggestion
def get_client():
   return docker.Client(base_url="unix://var/run/docker.sock")
```
From https://docker-py.readthedocs.io/en/1.6.0/api/

This should be in a setup fixture

File: test/dockerpy/images.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+class TestImages(unittest.TestCase):
+    client = ""
+    def test_get_image(self):
+        client = get_client()
+        bb = client.images.get("busybox")
+        self.assertTrue(busybox in bb.tags)
```
Comment (jwhonce) [inline test/dockerpy/images.py:14]:
```suggestion
        self.assertTrue(busybox in bb.RepoTags)
```
From https://docker-py.readthedocs.io/en/1.6.0/api/#images

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming the tests get happy.

**Comment by rhatdan** [comment]
The damn merge button on github causes the gating test to fail.

**Pull request closed without merging**